tools/flakeytests/reporter_test.go

```diff
@@ -23,8 +23,8 @@ func TestMakeRequest_SingleTest(t *testing.T) {
 	assert.Len(t, pr.Streams, 1)
 	assert.Equal(t, pr.Streams[0].Stream, map[string]string{"command": "go_core_tests", "app": "flakey-test-reporter"})
 	assert.ElementsMatch(t, pr.Streams[0].Values, [][]string{
-		{ts, "{\"package\":\"core/assets\",\"test_name\":\"TestLink\",\"fq_test_name\":\"core/assets:TestLink\"}"},
-		{ts, "{\"num_flakes\":1}"},
+		{ts, "{\"package\":\"core/assets\",\"test_name\":\"TestLink\",\"fq_test_name\":\"core/assets:TestLink\",\"commit_sha\":\"\",\"repository\":\"\",\"event_type\":\"\"}"},
+		{ts, "{\"num_flakes\":1,\"commit_sha\":\"\",\"repository\":\"\",\"event_type\":\"\"}"},
```

Comment:
nit/ back-quoted raw strings might be nice given all of this quote escaping,

Reply:
Good point! I've changed this to using back-quotes instead.